pkg/prowgen/prowgen.go

```diff
@@ -125,6 +126,11 @@ func GenerateJobs(configSpec *cioperatorapi.ReleaseBuildConfiguration, info *Pro
 	if release, found := configSpec.Releases[cioperatorapi.LatestReleaseName]; found && release.Candidate != nil {
 		jobRelease = release.Candidate.Version
 	}
+
+	skipCloning := true
+	if configSpec.BuildRootImage != nil && configSpec.BuildRootImage.FromRepository {
```

Comment:
Is it worth it to try to make this shallow, too, to save on time?

Reply:
I don't see an option to enable shallow cloning in the decoration utils